osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -795,20 +841,14 @@ namespace renderkit {
         }
         checkForGLError("RenderManagerOpenGL::RenderDisplayInitialize end");
 
-		// Store the frame buffer that was active before we started rendering,
-		// so we can put it back when we finalize.
-		GLint fb;
-		glGetIntegerv(GL_FRAMEBUFFER_BINDING, &fb);
-		m_initialFrameBuffer = static_cast<GLuint>(fb);
-
 		return true;
     }
 
     bool RenderManagerOpenGL::RenderEyeInitialize(size_t eye) {
         checkForGLError("RenderManagerOpenGL::RenderEyeInitialize starting");
 
         // Render to our framebuffer
-        glBindFramebuffer(GL_FRAMEBUFFER, m_frameBuffers.at(GetDisplayUsedByEye(eye)));
+        glBindFramebuffer(GL_FRAMEBUFFER, GetDisplayUsedByEye(eye));
```

Comment:
This seems like it is removing a level of indirection that was put in on purpose.  It may work in some cases, but this lookup is not a performance hit so why not put it back the way it was?

Reply:
I've already reverted this change.